frontend/src/components/MastodonLogo.tsx

```diff
@@ -1,96 +1,81 @@
 import { component$ } from '@builder.io/qwik'
 
 export const heightsMap = {
-	small: '2rem',
-	medium: '2.65rem',
+	small: '2.4rem',
+	medium: '3.5rem',
+	large: '6rem',
 } as const
 
 type Props = {
 	size: keyof typeof heightsMap
 }
 
-// TODO: the mastodonLogo is likely no longer needed
-//       we're keeping it around just in case but before
-//       launch this should be removed
-export const MastodonLogo = component$<Props>(({ size }) => {
```

Comment:
@jculvey since I was tweaking the heightsMap values I figured I could also just remove the Mastodon logo as I imagine we aren't going to need it at this point (and it may look off with the new height values)